### PR TITLE
[WebAPI] TC for DynamicsCompressor.reduction for Chromium 52

### PR DESCRIPTION
**Pull request by renchenglei**
Impacted tests(approved): new 1, update 1, delete 0
Unit test platform: Crosswalk for Android 22.52.557.0
Unit test result summary: pass 2, fail 0, block 0

https://crosswalk-project.org/jira/browse/CTS-1851

### CODE REVIEW

**Comment by renchenglei** [comment]
@Honry PTAK, Thanks a lot!

File: webapi/tct-webaudio-w3c-tests/webaudio/dynamicscompressornode_reduction_default_value.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+Authors:
+        Ren, Chenglei <chenglei.ren@intel.com>
+-->
+
+<html>
+  <head>
+    <title>WebAudio Test: dynamicscompressornode_reduction_default_value</title>
```
Comment (Honry) [inline webapi/tct-webaudio-w3c-tests/webaudio/dynamicscompressornode_reduction_default_value.html]: Please follow the latest template, https://github.com/crosswalk-project/crosswalk-test-suite/blob/master/tools/template/script_test_case.html

**Comment by renchenglei** [comment]
@Honry I have modified the code, ptal, thanks!

**Comment by Honry** [comment]
LGTM

**Pull request merged**